tests/test_memory.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import gdb
+import pytest
+
+import pwndbg.memory
+import pwndbg.stack
+
+
+def test_memory_read_write(entry_binary):
+    """
+    Tests simple pwndbg's memory read/write operations with different argument types
+    """
+    entry_binary('reference-binary.out')
+    stack_addr = next(iter(pwndbg.stack.stacks.values())).vaddr
```

Comment:
Probably easier to just use `pwndbg.regs.stack`?

Reply:
MMm `pwndbg.regs.stack` returns a string e.g. of `esp` or `rsp`

Reply:
We could do `getattr(pwndbg.regs, pwndbg.regs.stack)` but I am not sure its the best way...